test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -148,30 +136,30 @@ TEST_F(SynchronizerTest, ValidWhenBadStorage) {
 
 TEST_F(SynchronizerTest, ValidWhenBlockValidationFailure) {
   // commit from consensus => chain validation failed => commit successful
-  Block test_block;
-  test_block.height = 5;
-  test_block.sigs.emplace_back();
-  test_block.hash = iroha::hash(test_block);
+  std::shared_ptr<shared_model::interface::Block> test_block =
+      std::make_shared<shared_model::proto::Block>(
+          shared_model::proto::TemplateBlockBuilder<
```

Comment:
I understand why do you do that kind of definition of the builder. But better approach is to create TestUnsignedBlockBuilder. Take a look at `test_query_builder.hpp`:
https://github.com/hyperledger/iroha/blob/1adfd054d226c27d3ef0ba341aba6489766a63b1/test/module/shared_model/builders/protobuf/test_query_builder.hpp#L33-L36